docs/diagrams/CommandUML.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+alt add patient
+    Manager -> Command: addPatient
+    Command -> PatientList: add
+else delete patient
+    Manager -> Command: deletePatient
```

Comment:
Suggest to add a notation to declare that this is  a minimum notation sequence diagram.